fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/ProgressBarTokens.kt

```diff
@@ -0,0 +1,110 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.*
+import kotlinx.parcelize.Parcelize
+
+enum class ProgressBarType{
+    LinearProgressBar,
+    CircularProgressBar,
+    Shimmer
+}
+enum class CircularProgressBarIndicatorSize{
+    XXSmall,
+    XSmall,
+    Medium,
+    Large,
+    XLarge
+}
+enum class LinearProgressBarHeight{
+    XXXSmall
+}
+enum class ShimmerShape {
+    Box,
+    Circle
+}
+data class ProgressBarInfo(
+    val progressBarType: ProgressBarType = ProgressBarType.LinearProgressBar,
+    val circularProgressBarIndicatorSize: CircularProgressBarIndicatorSize = CircularProgressBarIndicatorSize.XSmall,
+    val linearProgressBarHeight: LinearProgressBarHeight = LinearProgressBarHeight.XXXSmall,
+    val neutralColor: Boolean = true
+): ControlInfo
+
+@Parcelize
+open class ProgressBarTokens: ControlToken, Parcelable{
```

Comment:
As the API are different for ProgressBar, hence it is better to have different token class for each. Please break it down.